[2025-10-07T10:17:22.796+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:17:22.870+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=27 [queued]>
[2025-10-07T10:17:22.907+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=27 [queued]>
[2025-10-07T10:17:22.915+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:17:23.436+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:16:29.686945+00:00
[2025-10-07T10:17:23.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=980) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:17:23.476+0000] {standard_task_runner.py:72} INFO - Started process 1148 to run task
[2025-10-07T10:17:23.480+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:16:29.686945+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp0ggzalwv', '--map-index', '27']
[2025-10-07T10:17:23.490+0000] {standard_task_runner.py:105} INFO - Job 68: Subtask process_batch
[2025-10-07T10:17:23.551+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:17:24.573+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:16:29.686945+00:00 map_index=27 [running]> on host e3278e0f0bbc
[2025-10-07T10:17:25.303+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:16:29.686945+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:16:29.686945+00:00'
[2025-10-07T10:17:25.307+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:17:25.312+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DMC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.314+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DM7: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.315+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DLT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.316+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DLR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.317+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DLM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.318+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DLG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.319+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DLD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.320+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DL1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.321+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DKW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.321+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DKH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.322+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DKG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.323+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DKC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.325+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DIH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.326+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DIG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.328+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DID: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.328+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DIC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.331+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.331+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.332+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.333+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.333+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.334+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.335+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.336+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.336+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DHA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.337+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DGW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.338+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DGT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.338+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DGC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.339+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DFF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.343+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DFC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.346+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DDV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.347+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DDN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.349+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DDM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.350+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DDH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.353+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DDG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.354+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DDB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.360+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.361+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.361+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.362+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.363+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.369+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.369+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.371+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DCF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.371+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DC4: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.373+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DC2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.376+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DC1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.378+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DBW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.380+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DBT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.381+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý DBM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:17:25.381+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:17:25.419+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:17:25.420+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:16:29.686945+00:00, map_index=27, execution_date=20251007T101629, start_date=20251007T101722, end_date=20251007T101725
[2025-10-07T10:17:25.763+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:17:26.386+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:17:26.395+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
